docs/src/content/docs/reference/config/pre-tool-use.md

```
@@ -172,6 +172,7 @@ Two formats are supported for backward compatibility:
 Allows providing a custom error message that will be shown when Claude attempts to edit a file matching this pattern.
 
    Properties:
+   - `agent` (string | null): Optional agent pattern to scope this rule to specific agents (e.g., "coder", "tester", "main", or glob patterns like "code*")
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)